packages/website/src/components/Modals/RecruitmentModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import {
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from '@coderscamp/ui/components/Modal';
+import { Typography } from '@coderscamp/ui/components/Typography';
+
+import { forMentors } from './modal.data';
+import { ModalForm } from './ModalForm';
+
+type RecruitmentModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  modalText?: {
+    header: string;
+    body: string;
+    footer: string;
+  };
+};
+
+export const RecruitmentModal = ({ isOpen, onClose, modalText = forMentors }: RecruitmentModalProps) => {
+  return (
+    <>
+      <Modal isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+
+        <ModalContent
+          p={{ base: '32px 16px 16px 32px', md: '56px 64px 64px 64px' }}
+          borderRadius="8px"
+          maxWidth={800}
+          width={800}
+          minHeight={448}
+          boxShadow="large"
+        >
+          <ModalHeader p={0}>
+            <Typography
+              as="h2"
+              size="4xl"
+              weight="extrabold"
+              color="gray.900"
+              textAlign="center"
```

Comment:
💭 As all texts are aligned to the center, you can set this prop only once for the `ModalContent` element